lib/hammer_cli_katello/repository.rb

```diff
@@ -567,6 +567,17 @@ class RemoveContentCommand < HammerCLIKatello::SingleResourceCommand
       end
     end
 
+    class RepublishCommand < HammerCLIKatello::SingleResourceCommand
+      action :republish
```

Comment:
Should we include HammerCLIForemanTasks::Async to this as this could take a while to finish for larger repos?

Reply:
Also extend RepositoryScopedToProduct in this?

Reply:
@sjha4 updated

Reply:
We'll need similar params and validations like the other commands in the file to allow repository name to be scoped to product and org etc? See reclaim-space command for example.. 
include OrganizationOptions
      validate_repo_name_requires_product_options

Reply:
@sjha4 updated